.github/workflows/php.yml

```diff
@@ -0,0 +1,64 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Similar to @mlahargou's comment above, does this mean we're only running this workflow on pull requests that we want to merge into master? If we have two different PRs with the second branching off of the first, and set the second PR to merge into the first PR's branch, will this run for the second PR?

Reply:
Good call 0c9e702